Specifications/model/information.model.md

```diff
@@ -0,0 +1,151 @@
+
+### Asset Transfer
+
+A set of interactions between a provider `Connector` and consumer `Connector` that give access to an `Asset` under the terms of an `Agreement`.
+
+## 2 Dataspace Information Model
+
+The following sections outline the Dataspace Information Model, which form the foundation of this specification.
+
+### 2.1 Dataspace Entity Relationships
+
+The relationships between the primary dataspace entities are defined as follows:
+
+![](./im.dataspace.relationships.png)
+
+Note that all relationships are multiplicities unless specified.
+
+- A `Dataspace Authority` manges one or more `Dataspaces`. This will include participant registration and may entail mandating business and/or requirements. For example, a
+  Dataspace Authority may require participants to obtain some form of business certification. A Dataspace authority may also impose technical requirements such as support for the
+  technical enforcements of specific usage policies.
+- A `Participant` is a member of one or more `Dataspaces`. A participant registers `Participant Agents` that perform tasks on its behalf.
+- A `Participant Agent` performs tasks such as publishing a catalog or engaging in an asset transfer. In order to accomplish these tasks, a participant agent may
+  use a _**verifiable presentation**_ generated from a _**credential**_ obtained from a third-party issuer. A participant agent may also use an _**ID token**_ issued by a
+  third-party identity provider. Note that a participant agent is a logical construct and does not necessarily correspond to a single runtime process.
+- An `Identity Provider` is a trust anchor that generates `ID tokens` used to verify the identity of a `Participant Agent`. Multiple identity providers may operate in
+  a dataspace. The identity standard used by a provider is not defined but could be _OAuth2_ or _Decentralized Identifiers using did:web_. An identity provider may be a third-party
+  or a participant itself (for example, in the case of decentralized identifiers).
+- A `Credential Issuer` issues _verifiable credentials_ used by participant agents to allow access to assets and verify usage control.
+
+The diagram below depicts the relationships between `ParticipantAgent` types:
+
+![](./im.participant.entities.png)
+
+- A `CatalogService` is a `Participant Agent` that makes a [DCAT Catalog](https://www.w3.org/TR/vocab-dcat-3/#Class:Catalog) available to other participants.
+- A `Catalog` contains one or more `Asset Entries`, which are [DCAT Datasets](https://www.w3.org/TR/vocab-dcat-3/#Class:Dataset). A `Catalog` also contains **_at least one_**
+  [DCAT DataService](https://www.w3.org/TR/vocab-dcat-3/#Class:Data_Service) that references a `Connector` where assets may be obtained.
+- An `Asset Entry` has **one** `Offer`, which is an [ODRL Offer](https://www.w3.org/TR/odrl-model/#policy-offer) describing the _usage control policy_ associated with the asset.
```

Comment:
We also need a mechanism to describe access policies on asset entries, something which is missing for a long time...

Reply:
I don't think this has to be part of these documents here as access policies on assets are not communicated via the protocol. However, I agree, this has to be mentioned in some connector architecture/functionality documents.

Reply:
I don't think this has to be part of these documents here as access policies on assets are not communicated via the protocol. However, I agree, this has to be mentioned in some connector architecture/functionality documents.